content/introduction/overview.md

```diff
@@ -30,57 +34,57 @@ Making it possible to download a file from many locations that aren’t managed
 
 - **Makes it hard for a website to go offline.** If someone attacks Wikipedia’s web servers or an engineer at Wikipedia makes a big mistake that causes their servers to catch fire, you can still get the same page from somewhere else.
 
-- **Makes it harder for authorities to censor content.** Because files on IPFS can come from many places and because some of those places might be nearby, it’s very hard for authorities (whether they’re states, corporations, or someone else) to block things. In 2017, Turkey blocked Wikipedia and Spain blocked access to Catalonian independence sites. We hope IPFS can prevent actions like those.
+- **Makes it harder for authorities to censor content.** Because files on IPFS can come from many places, it’s very hard for authorities (whether they’re states, corporations, or someone else) to block things. In 2017, Turkey blocked Wikipedia and Spain blocked access to Catalonian independence sites. We hope IPFS can prevent actions like those.
```

Comment:
To be precise, the IPFS network incorporates no censorship support. It is in fact very easy to block. It would be more accurate to say that ipfs provides new avenues for censorship resistant systems. We are working on this but we should be very clear that it does not provide censorship support or anonymity. Using ipfs for anti-censorship activities might make you a very clear target. 

`We hope IPFS can prevent actions like those` -> cannot prevent, but can provide ways to circumvent when they happen.

Note that  `Catalonian independence sites` is a generalization (sites were not blocked because for being around "independence", thus not all of those sites were blocked). Not sure if accuracy is needed though.

Reply:
Updated.